EIPS/eip-new.md

```diff
@@ -0,0 +1,124 @@
+    gasPrice: QUANTITY - (optional, default: To-Be-Determined) Integer of the gasPrice used for each paid gas, in Wei.
+    value: QUANTITY - (optional) Integer of the value sent with this transaction, in Wei.
+    data: DATA - The compiled code of a contract OR the hash of the invoked method signature and encoded parameters.
+    nonce: QUANTITY - (optional) Integer of a nonce. This allows to overwrite your own pending transactions that use the same nonce.
+  
+Returns
+  DATA, The signed transaction object.
+```
+
+#### wallet_silentSendTransaction
+This command will create a transaction without asking the user and send it to the blockchain.
+```
+Parameters
+  clientSecret: DATA, 20 Bytes - secret key obtained from game_requestAutoSign method
+  Object - The transaction object
+    from: DATA, 20 Bytes - The address the transaction is sent from.
+    to: DATA, 20 Bytes - (optional when creating new contract) The address the transaction is directed to.
+    gas: QUANTITY - (optional, default: 90000) Integer of the gas provided for the transaction execution. It will return unused gas.
+    gasPrice: QUANTITY - (optional, default: To-Be-Determined) Integer of the gasPrice used for each paid gas.
+    value: QUANTITY - (optional) Integer of the value sent with this transaction.
+    data: DATA - The compiled code of a contract OR the hash of the invoked method signature and encoded parameters.
+    nonce: QUANTITY - (optional) Integer of a nonce. This allows to overwrite your own pending transactions that use the same nonce.
+
+Returns
+  DATA, 32 Bytes - the transaction hash, or the zero hash if the transaction is not yet available.
+```
+
+### Application and Wallet Communication
+When the wallet application is closed or in the background, it cannot connect to the bridge server via WebSocket. So that we have to trigger the wallet to connect to the bridge and get waiting for requests. For this reason, the Push Notification method should be used. So only the wallets that have push notification support can implement this feature.[3] [4]
+
+![](https://docs.walletconnect.com/assets/images/call-request-af4f9f2385303a6cd381c35f3b13b665.png)
+
+Whenever the wallet receives a push message. It connects to the bridge server and gets the pending requests. If there is a silent signing request like “**wallet_silenSignTransaction**” or “**wallet_silentSendTransaction**” and the requesting client is allowed at that time, then the wallet will execute the request without any UI message or prompt.
+
+## Backwards Compatibility
+These new RPC methods are not changing the current ones and the push notification API is currently part of the WalletConnect specification. Everything in this proposal does not change anything for other applications and wallets.
+
+## Security Considerations
+This feature should only be used with user consent. So the responsibility of allowing this is on the user. Users can decide to use the application as usual.
```

Comment:
While not a blocker for draft, for something like this I would want to see a *much* more complete set of security risks evaluated/described before moving to review.